pkg/settings/setting.go

```diff
@@ -129,12 +125,33 @@ var (
 
 	Rke2DefaultVersion = NewSetting("rke2-default-version", "")
 	K3sDefaultVersion  = NewSetting("k3s-default-version", "")
+
+	// KubeconfigGenerateToken determines whether the UI will return a generate token with kubeconfigs.
+	// If set to false the kubeconfig will contain a command to login to Rancher.
+	KubeconfigGenerateToken = NewSetting("kubeconfig-generate-token", "true")
+
+	// Deprecated: On removal use kubeconfig-default-ttl-minutes for all kubeconfigs.
+	KubeconfigTokenTTLMinutes = NewSetting("kubeconfig-token-ttl-minutes", "960") // 16 hours
+
+	// KubeconfigDefaultTokenTTLMinutes is the default time to live applied to kubeconfigs created for users.
```

Comment:
```suggestion
	// KubeconfigDefaultTokenTTLMinutes is the default time to live applied to kubeconfigs created for users, and it will take effect regardless of the kubeconfig-generate-token status.
```